src/components/NewChapterForm.tsx

```diff
@@ -0,0 +1,269 @@
+                />
+              </div>
+            </div>
+            <div className="w-full">
+              <div>Last Name</div>
+              <input
+                {...register("lastName")}
+                className="h-8 w-full rounded-md px-1 text-black"
+              />
+              <div className="text-sm">
+                <ErrorMessage
+                  errors={errors}
+                  name="lastName"
+                  render={({ message }) => (
+                    <p>
+                      <span className="text-sunset-orange">* </span>
+                      {message}
+                    </p>
+                  )}
+                />
+              </div>
+            </div>
+            <div className="w-full">
+              <div>University Email</div>
+              <input
+                {...register("universityEmail")}
+                className="h-8 w-full rounded-md px-1 text-black"
+              />
+              <div className="text-sm">
+                <ErrorMessage
+                  errors={errors}
+                  name="universityEmail"
+                  render={({ message }) => (
+                    <p>
+                      <span className="text-sunset-orange">* </span>
+                      {message}
+                    </p>
+                  )}
+                />
+              </div>
+            </div>
+            <div className="w-full">
+              <div>Phone Number</div>
+              <input
+                {...register("phoneNumber")}
+                className="h-8 w-full rounded-md px-1 text-black"
+              />
+              <div className="text-sm">
+                <ErrorMessage
+                  errors={errors}
+                  name="phoneNumber"
+                  render={({ message }) => (
+                    <p>
+                      <span className="text-sunset-orange">* </span>
+                      {message}
+                    </p>
+                  )}
+                />
+              </div>
+            </div>
+            <div className="w-full">
+              <div>College / University</div>
+              <input
+                {...register("university")}
+                className="h-8 w-full rounded-md px-1 text-black"
+              />
+              <div className="text-sm">
+                <ErrorMessage
+                  errors={errors}
+                  name="university"
+                  render={({ message }) => (
+                    <p>
+                      <span className="text-sunset-orange">* </span>
+                      {message}
+                    </p>
+                  )}
+                />
+              </div>
+            </div>
+            <div className="w-full">
+              <div>College / University Address</div>
+              <input
+                {...register("universityAddress")}
+                className="h-8 w-full rounded-md px-1 text-black"
+              />
+              <div className="text-sm">
```

Comment:
For readability, I would remove the `<div className="text-sm">...</div>`. Instead, I would move the text size into the `ErrorMessage` component.